tcf_website/migrations/0012_alter_review_email.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.8 on 2024-03-27 07:15
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('tcf_website', '0011_review_email'),
+    ]
+
+    operations = [
```

Comment:
We should merge these two migrations into one.

Reply:
Seems like you shouldn't delete and merge migrations since they're like GitHub commits, and the only way to merge would be to delete and remake them.
https://stackoverflow.com/questions/28404461/can-i-delete-the-django-migration-files-inside-migrations-directory
I don't think it changes efficiency much to have both migrations, since they're only ever run once.